index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Asap:wght@500&family=Poppins:wght@100;800&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
- [ ] Kindly add to your font weights e.g. 300, 400, 500, so you can make your paragraphs and headers with a smaller weight, to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/158728184-3549ec05-a616-46fd-897e-5849b5e01678.png) | ![image](https://user-images.githubusercontent.com/35535130/158728260-1881cb43-498f-4242-bea3-0ae48b12edfc.png) |